src/components/LeafletMap.js

```diff
@@ -233,6 +233,30 @@ class LeafletMap extends Component {
     return false;
   }
 
+  toFixedNoRounding = function(number, n) {
+    const reg = new RegExp('^-?\\d+(?:\\.\\d{0,' + n + '})?', 'g');
```

Comment:
Can you also add a comment above explaining what you are attempting to capture in this group? Would you also change the name `reg` to something such as `captureThisThingReg` (where thisThing is what you're trying to capture)